server/block/slime.go

```diff
@@ -0,0 +1,21 @@
+package block
+
+// A Slime block is a translucent block that entities can bounce on
+type Slime struct {
+	solid
+}
+
+// BreakInfo ...
+func (s Slime) BreakInfo() BreakInfo {
+	return newBreakInfo(0.1, alwaysHarvestable, shovelEffective, oneOf(s))
```

Comment:
i don't think there is any specific tool that's effective for breaking slimes?

Reply:
With the hand the block is one shot

Reply:
yea, but the it says here the tool to harvest slime blocks is a shovel

Reply:
done